projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.html

```diff
@@ -23,7 +23,7 @@
               [ngClass]="commentStyle()"
               [(ngModel)]="fullComment"
               [readOnly]="!editable"
-              (ngModelChange)="reRenderComments()"
+              (ngModelChange)="reRenderComments(); onCommentChange($event);"
```

Comment:
should this method not be on when the user clicks save? this will be called every time the user is typing.